.ci.yaml

```diff
@@ -121,7 +121,6 @@ platform_properties:
         ]
       os: Mac-12|Mac-13
       device_type: none
-      cpu: x86
```

Comment:
This makes `mac_x64` the same as `mac`. Switching these `mac_x64` targets to `mac`?

Reply:
Never mind. I missed the lastest comment.

Considering we have only 5 `mac_x64` targets, do we want to led to validate they pass to avoid potential revert if this causes tree red? We are missing them in pre-submit.

Or alternatively, we can update those `mac_x64` targets to `mac` with `brigup: true`, which is more safer.

Reply:
I'll send a second PR for that once we ensure all the tests pass correctly and we don't need to revert.

Reply:
I guess my point here is: do we want to land this PR and revert if it causes tree red, or do we want to validate them in staging pool without potentially breaking the tree. I would prefer the former option.

We have been doing similarly when switching targets from `linux_android` to `linux_pixel_7pro`:
https://github.com/flutter/flutter/pull/140389/files
https://github.com/flutter/flutter/pull/140438/files

Reply:
Updated them to use bringup.